minigalaxy/ui/gametile.py

```diff
@@ -609,7 +614,7 @@ def __state_uninstalling(self):
     def __state_updatable(self):
         self.update_icon.show()
         self.update_icon.set_from_icon_name("emblem-synchronizing", Gtk.IconSize.LARGE_TOOLBAR)
-        self.button.set_label(_("play"))
+        self.button.set_label(_("Play"))
```

Comment:
This change makes it necessary to update all translation files. So we'd either have to leave this change out or update those.